code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm

```diff
@@ -137,7 +137,7 @@
 	if(!.)
 		return FALSE
 
-	if(!A || A.layer >= FLY_LAYER)
+	if(!A || A.layer >= ABOVE_FLY_LAYER)	//Rouny can make the jump, they don't skip leg day
```

Comment:
explain this change

Reply:
So that Runners can pounce on flying things. More counters to aircraft is good.